.github/PULL_REQUEST_TEMPLATE/PR_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Thank you for contributing. You're awesome!
+
+When you have submitted the request don't forget to ask for a review. We may respond with some clarifying questions or request minor changes before approval and merge.
+
+Please clearly fill out all the information below.
+
+### What
+##### In words what does this PR address (bug fix, feature, etc.)?
+
+### How
+##### Describe the current behavior, what the new behavior will be, and how you achieved your results.
+
+### Linked Tickets
+##### Link the tickets that this PR addresses here.
```

Comment:
1. Pull requests need not have a linked issue. There are too many things people can PR to make a ticket for each.
2. GitHub has a feature that can auto-mark issues as closed when a relevant pull request is closed. I believe this happens when key words like `closes` and `resolves` are used before the linked issue.

Reply:
Sweet. Thank you.